### PR TITLE
Initial proposal for how we document our output

### PR DESCRIPTION
**Pull request by joewalker**

### CODE REVIEW

File: text/0001-documenting-output.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  * Use [this template](../0000-template.md)
+3. As there are updates we keep the Markdown up to date
+4. We may publish to a wider audience using a blog post or GitBook as required
+
+
+# Drawbacks / Unresolved Questions
+
+It is possible that GDocs has such a low update friction that people will resent the PR process imposed by Github. If this turns out to be true then we will need to update this process.
+
+This process is somewhat modelled on the [Rust RTFC](https://github.com/rust-lang/rfcs) process. It isn't certain that we need the numeric prefix.
```
Comment (spinda) [inline text/0001-documenting-output.md]: `Rust RTFC` -> `Rust RFC`?

Comment (mykmelez) [reply text/0001-documenting-output.md]: Yep.

**Comment by mykmelez** [comment]
@joewalker I added clarified the reference to "browser-arch" and added links to the Google Group, IRC channel, and Slack channel. I also fixed the typo that @spinda noted. Besides those issues, this looks good to me, so I'll merge it. Let's give the process a try and see how it goes.

**Comment by joewalker** [comment]
@mykmelez @spinda Thanks for finishing this off.

**Pull request merged**